development.md

```diff
@@ -27,12 +27,11 @@ Note that the workbench can be also bundled independently to serve from some CDN
 
 ### Bundling
 
-Run `yarn gulp server-min` to create production-ready distributable from sources. After the build is finished, you will be able to find following folders next to the project directory:
-- `server-pkg-web` contains the web workbench static resources,
-- `server-pkg-server` contains the headless remote server with the remote CLI,
-- `serfver-pkg` contains everything together to be distributed standalone.
+Run `yarn gulp vscode-reh-web-linux-x64-min` to create production-ready distributable from sources. After the build is finished, you will be able to find the `vscode-reh-web-linux-x64` folder next to the repository folder (one level up from where you executed the command). In this folder you will find `server.sh`: your entrypoint to the OpenVSCode server.
```

Comment:
`server.sh` is deprecated now the entrypoint is inside bin folder: `bin/openvscode-server`

Reply:
Thanks for letting me know, I will fix this in https://github.com/gitpod-io/openvscode-releases as well.

Reply:
@jeanp413 it looks like this has already been done for the releases Dockerfile in https://github.com/gitpod-io/openvscode-releases/commit/5b1eb21b016b5f4c57d15730cae62951db5faf0d#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L60